deployments/sgx_epc_extendedresource/daemonset.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: extres
+  namespace: kube-system
+  labels:
+    app: extended-resource-example
+spec:
+  selector:
+    matchLabels:
+      app: extended-resource-example
+  template:
+    metadata:
+      labels:
+        app: extended-resource-example
+    spec:
+      serviceAccountName: sgx-epc-extres-api
+      initContainers:
+      - name: sgx-epc-extended-resource
+        image: sgx-epc-extended-resource:0.1
+        imagePullPolicy: Never
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
```

Comment:
would `kustomize` overlay patch to change the existing `initContainers` in the sgx plugin deployment be an option for you?

Reply:
Now we don't need those files